biicode.conf

```diff
@@ -0,0 +1,4 @@
+[parent]
+
+[paths]
+	include
```

Comment:
What is this file for?  Could you add comments?

Would it be possible to put everything related to biicode in a subdirectory for example?